bftengine/src/bftengine/ReplicaImp.hpp

```diff
@@ -682,6 +682,7 @@ class ReplicaImp : public InternalReplicaApi, public ReplicaForStateTransfer {
   batchingLogic::RequestsBatchingLogic reqBatchingLogic_;
   ReplicaStatusHandlers replStatusHandlers_;
   concord::util::CallbackRegistry<uint64_t> onViewNumCallbacks_;
+  concord::util::CallbackRegistry<SeqNum> createDbCheckpointCb_;
```

Comment:
@nkumar04, my comment was to introduce
`concord::util::CallbackRegistry<SeqNum> onSeqNumIsStableCallbacks_;`
and eventually to move all the callbacks from `ReplicaImp::onSeqNumIsStable()` to it.
And we don't need a `createDbCheckpointCb_`.

Reply:
fixed